include/CppUTestExt/MockSupport.h

```diff
@@ -135,7 +135,8 @@ class MockSupport
     MockExpectedCallComposite compositeCalls_;
     MockNamedValueComparatorsAndCopiersRepository comparatorsAndCopiersRepository_;
     MockNamedValueList data_;
-
+    SimpleString scope_;
```

Comment:
yes, I can't easily get the name from GetData, previously global mock has one uni-direction association to mockNamedValueNode which own scope mock and it's name. the data_ in scope mock is initialized as empty so getData in scope mock would return empty.

but you are right, scope name can be filled to the mockNamedValue list, the data_ member.

Let's say my updated commit for this proposl!